src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.cs

```diff
@@ -80,31 +82,314 @@ public IList<MenuFlyoutItemBase> Items
 
 		#endregion
 
-		public global::Windows.UI.Xaml.Style MenuFlyoutPresenterStyle
+		public Style MenuFlyoutPresenterStyle
 		{
 			get => (Style)this.GetValue(MenuFlyoutPresenterStyleProperty);
 			set => SetValue(MenuFlyoutPresenterStyleProperty, value);
 		}
 
-		public static global::Windows.UI.Xaml.DependencyProperty MenuFlyoutPresenterStyleProperty { get; } =
-		Windows.UI.Xaml.DependencyProperty.Register(
+		public static DependencyProperty MenuFlyoutPresenterStyleProperty { get; } =
+		DependencyProperty.Register(
 			"MenuFlyoutPresenterStyle",
-			typeof(global::Windows.UI.Xaml.Style),
-			typeof(global::Windows.UI.Xaml.Controls.MenuFlyout),
-			new FrameworkPropertyMetadata(null));
+			typeof(Style),
+			typeof(MenuFlyout),
+			new PropertyMetadata(null, (s, e) => (s as MenuFlyout).OnPropertyChanged2(s, e)));
+
+		public void ShowAt(UIElement targetElement, Point point)
+		{
+			ShowAtCore((FrameworkElement)targetElement);
+		}
 
-		public void ShowAt(global::Windows.UI.Xaml.UIElement targetElement, global::Windows.Foundation.Point point)
+		private void OnPropertyChanged2(DependencyObject s, DependencyPropertyChangedEventArgs e)
 		{
-			base.ShowAt((FrameworkElement)targetElement);
+			if (e.Property == MenuFlyoutPresenterStyleProperty &&
+				GetPresenter() != null)
+			{
+				SetPresenterStyle(GetPresenter(), e.NewValue as Style);
+			}
 		}
 
 		protected override Control CreatePresenter()
 		{
-			_presenter = new MenuFlyoutPresenter();
+			MenuFlyoutPresenter presenter = new MenuFlyoutPresenter();
+			Style style;
+
+			presenter.SetParentMenuFlyout(this);
+
+			style = MenuFlyoutPresenterStyle;
+			SetPresenterStyle(presenter, style);
+
+			return presenter;
+		}
+
+		private protected override void ShowAtCore(FrameworkElement pPlacementTarget)
+		{
+			base.ShowAtCore(pPlacementTarget);
+
+			if (m_openWindowed)
+			{
+				// UNO TODO
+				// SetIsWindowedPopup();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)